data/transactions/logic/eval.go

```diff
@@ -199,7 +199,7 @@ type LedgerForLogic interface {
 	AccountData(addr basics.Address) (ledgercore.AccountData, error)
 	Authorizer(addr basics.Address) (basics.Address, error)
 	Round() basics.Round
-	LatestTimestamp() int64
+	PrevTimestamp() int64
```

Comment:
@jannotti Double checking myself:  Is it by design to remove the following interface methods from `master`?

```
	LatestTimestamp() int64
	BlockHdrCached(basics.Round) (bookkeeping.BlockHeader, error)
```

I'm asking without having done extensive research.  I went so far as to recall `BlockHdrCached` is introduced by https://github.com/algorand/go-algorand/pull/3900.

Reply:
`LatestTimestamp` will be removed from `LedgerForLogic` but will simply become `PrevTimestamp`, to match what it's called elsewhere. (It was a wrapper that just changed the name.)

`BlockHdrCached` is in master now, and will stay, until/unless we unify the two BlockHeader access functions. Only `BlockHdrCached` should be used in the AVM, so it is in `LedgerForLogic` while `BlockHdr` remains in the main Ledger interface.

Reply:
@jannotti Resolving - Thanks for the reply.  Sorry for the confusion.  I commented in the wrong PR while comparing diffs.